tests/test_vlan_stacking.py

```diff
@@ -0,0 +1,199 @@
+import pytest
```

Comment:
@prabhataravind , please review the test/cleanup

Reply:
Test/cleanup Looks good to me!